app/assets/stylesheets/common/_constants.scss

```diff
@@ -1,6 +1,6 @@
            $fonts: 'Open Sans', 'Lucida Grande', Verdana, Arial, sans-serif;
       $mono-fonts: 'Consolas', 'Courier New', Courier, mono, serif;
-          $radius: 5px;

```

Comment:
Preference: Markus looks better when it is angular and has sharper, square edges.